### PR TITLE
pipewire: mention xdg autostart files

### PR DESCRIPTION
**Pull request by paper42**
needs [void-packages#37822](https://github.com/void-linux/void-packages/pull/37822) to be merged first

### CODE REVIEW

File: src/config/media/pipewire.md [hunk 1]
```diff
@@ -14,6 +14,15 @@ environment or [startx](../graphical-session/xorg.md#startx). The `pipewire`
 package provides `pipewire` and `pipewire-pulse` system services, but they are
 not recommended for a typical setup.
 
+The `pipewire` package ships XDG Autostart files for `pipewire` and
+`pipewire-pulse` in `/usr/share/doc/pipewire`. If your desktop environment
+supports the XDG Autostart specification, you can enable pipewire by symlinking
+the files from documentation to the XDG autostart directory:
+
+```
+#: ln -s /usr/share/pipewire/pipewire.desktop /etc/xdg/autostart/pipewire.desktop
```
Comment (classabbyamp) [inline src/config/media/pipewire.md:23]:
```suggestion
# ln -s /usr/share/pipewire/pipewire.desktop /etc/xdg/autostart/pipewire.desktop
```
is the usual style I think

Comment (paper42) [reply src/config/media/pipewire.md]: thanks, fixed

**Comment by st3r4g** [comment]
Maybe also mention it can be done per user with:
```
$ ln -s /usr/share/applications/pipewire.desktop ~/.config/autostart
```

**Comment by paper42** [comment]
> Maybe also mention it can be done per user with:
> 
> ```
> $ ln -s /usr/share/applications/pipewire.desktop ~/.config/autostart
> ```

I can't think of a common usecase where enabling autostart per-user would make sense. If there is one, we can mention it, but people can probably figure it out if they want something less usual.

**Pull request merged**